libbpf-tools/memleak.c

```diff
@@ -283,14 +293,10 @@ int main(int argc, char *argv[])
 		return 1;
 	}
 
-	if (!strlen(env.object)) {
-		printf("using default object: %s\n", default_object);
+	if (!strlen(env.object))
 		strncpy(env.object, default_object, sizeof(env.object) - 1);
-	}
 
 	env.page_size = sysconf(_SC_PAGE_SIZE);
-	printf("using page size: %ld\n", env.page_size);
```

Comment:
This information is only valid when the -p option is not used and should be modified to be printed only in that case if needed.
As its necessity is unclear and to align with memleak Python's header format, it's removed.
Please provide feedback if necessary.